core-services/prow/02_config/_plugins.yaml

```diff
@@ -343,6 +343,12 @@ config_updater:
         - ci
         - ci-stg
       name: ci-operator-misc-configs
+    ci-operator/config/integr8ly/*/*-v*.*.yaml:
```

Comment:
That will break a lot of stuff. Why do you need to add this? Can you specify which file you are trying to include and let's find a better way?

Reply:
Yeah this might be unfortunate to hit on components that have "words" starting with `v`...

Reply:
One thing that will make this less likely to pick up configurations that other teams write is if you prefix the globs with your specific org/repo -- see the cluster logging operator globs below.

Reply:
Thanks for the feedback. Wasn't 100% sure of this change tbh.

I assumed since i was prefixing the globs already with ci-operator/config/integr8ly it wouldn't be able to affect anything outside of our own repos.

Would prefixing like this work better?
```
ci-operator/config/integr8ly/installation/integr8ly-installation-v*.*.yaml
```

To pick up on a file called:

```
ci-operator/config/integr8ly/installation/integr8ly-installation-v1.6.yaml
```

I am correct in thinking this file needs to be named with the correct branch name right? To start getting tests for PRs going back to a branch named v1.6 we would need add this separate config?

Reply:
Since its only 1 file, let's add it directly and try not to mess up with different regexes.